src/init/documentHandlers.ts

```diff
@@ -25,10 +26,12 @@ export function initializeDocumentHandlers() {
   };
 
   highlights.init(highlighter);
+  chrome.runtime.onMessage.addListener(contextMenuAnnotateCallback);
```

Comment:
maybe it'd worth to hide it behind an abstraction?

Reply:
I reflected it might be good to keep the code "lean" sometimes and expose these details -- especially for people who are not very accustomed to browser API (which is everybody here except me at the very moment of writing it)
But yes, I agree it's worth changing at the first opportunity